lib/import-export-visitor.js

```diff
@@ -403,7 +423,7 @@ IEVp.visitExportDefaultDeclaration = function (path) {
     // like a comma-separated sequence expression.
     this.overwrite(decl.start, dd.start, exportDefaultPrefix);
 
-    path.call(this.visitWithoutReset, "declaration");
+    path.call((p) => this.visitWithoutReset(p), "declaration");
```

Comment:
`this.visitWithoutReset` is already bound to `this`, so this new arrow function is unnecessary.